bin/scripts/syncWithProject.js

```diff
@@ -0,0 +1,25 @@
+const { TscWatchClient } = require('tsc-watch/client');
+const { execSync } = require('child_process');
+const project = process.argv[2];
+
+const watch = new TscWatchClient();
+
+if (typeof project !== 'string') {
+  throw new Error('Expect string for argument project');
+}
+
+const syncChanges = () => {
+  console.log('Copying changes to', project);
+  execSync('yarn copyAdditionalFiles && yarn buildDeclaredStyles', {
+    stdio: 'inherit',
+  });
+  execSync(
+    `cp -r ./dist/ ../${project}/node_modules/@lifeomic/react-native-sdk`,
+    {
+      stdio: 'inherit',
+    },
+  );
+  console.log('Copy complete');
+};
+watch.start('--project', 'tsconfig.build.json', '--outDir', 'dist');
```

Comment:
Nice so does this build it and then sync it into another projects node_modules or do I need to run the build separately?

Reply:
This handles building and copying so no other commands are needed.